scheduler/postgresql/sql/init_cook_database.sql

```diff
@@ -0,0 +1,42 @@
+-- Initialize a cook database from scratch --- creating the schemas and such.
+-- Assumes we already have an appropriately configured postgresql database and
+-- have psql connected to it.
+
+-- TODO: LOOK AT LIQUIDBASE FOR SETTING UP SQL
+BEGIN TRANSACTION;
+-- Always run this in the transaction so that if the set schema fails for any reason, we abort instead of possibly writing to the wrong schema's tables.
+CREATE SCHEMA :cook_schema;
+SET SCHEMA :'cook_schema';
+
+-- If you get a crazy error where the two above lines pass, but 'No schema has been selected to
+-- create in' when running the first CREATE TABLE. It can be caused by there being a capital
+-- letter in cook_schema. Schema names are lowercased when created, but case-sensitive when
+-- in the search path.
+
+--- Setup the tables.
+create table resources (
+   resource_name varchar(30) PRIMARY KEY,
+   resource_description text
```

Comment:
Should we make this `NOT NULL`?

Reply:
It wasn't clear to me from the file that this was "currently dead" -- should we add a comment in the file explaining what you explained above?

Reply:
Better idea. I'll just strip it from this review.